java/feathub-connectors/flink-connectors/flink-connector-prometheus/src/main/java/com/alibaba/feathub/flink/connectors/prometheus/PrometheusConfigs.java

```diff
@@ -58,4 +58,11 @@ public class PrometheusConfigs {
                                     + "and value are separated by '=', and labels are separated by ',', e.g., "
                                     + "k1=v1,k2=v2. Please ensure that your grouping key meets the Prometheus requirements. "
                                     + "Use backslash '\\' to escape '=' and ',' if they are used in label value.");
+
+    public static final ConfigOption<Long> RETRY_TIMEOUT_MS =
+            ConfigOptions.key("retryTimeoutMs")
+                    .longType()
+                    .defaultValue(0L)
+                    .withDescription(
+                            "The max timeout in milliseconds this sink may take retrying in case of occasional exceptions.");
```

Comment:
Let's mention that the metric will drop in case of a retry timeout, instead of failing the job.